diff --git a/node_modules/@electron/notarize/lib/check-signature.js b/node_modules/@electron/notarize/lib/check-signature.js
index 324568a..dd16722 100644
--- a/node_modules/@electron/notarize/lib/check-signature.js
+++ b/node_modules/@electron/notarize/lib/check-signature.js
@@ -41,14 +41,14 @@ const spawn_1 = require("./spawn");
 const debug_1 = __importDefault(require("debug"));
 const d = (0, debug_1.default)('electron-notarize');
 const codesignDisplay = (opts) => __awaiter(void 0, void 0, void 0, function* () {
-    const result = yield (0, spawn_1.spawn)('codesign', ['-dv', '-vvvv', '--deep', path.basename(opts.appPath)], {
+    const result = yield (0, spawn_1.spawn)('codesign', ['-dv', '-vvvv', '--deep', opts.appPath], {
         cwd: path.dirname(opts.appPath),
     });
     return result;
 });
 const codesign = (opts) => __awaiter(void 0, void 0, void 0, function* () {
     d('attempting to check codesign of app:', opts.appPath);
-    const result = yield (0, spawn_1.spawn)('codesign', ['-vvv', '--deep', '--strict', path.basename(opts.appPath)], {
+    const result = yield (0, spawn_1.spawn)('codesign', ['-vvv', '--deep', '--strict', opts.appPath], {
         cwd: path.dirname(opts.appPath),
     });
     return result;
